### PR TITLE
refactor(event): create eventlist serializer with fewer fields

### LINKED ISSUE #248: Fjern listen over påmeldte fra responsen fra /events og /events/<event_id>/
Feltet med listen over påmeldte (`registered_users_list`) er ikke nødvendig og skal fjernes.

### PR DESCRIPTION
**Pull request by olros**
## Proposed changes
Removed `registered_users_list` and `is_user_registered`-fields as they're not used in frontend. Created a ListSerializer with fewer fields to reduce the amount of data sent when fetching a list of events

Issue number: closes #248 

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] CHANGELOG.md has been updated. [Guide](https://tihlde.slab.com/posts/changelog-z8hybjom)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Pull request title follows [conventional commits](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) (`type(scope): description`)
- [x] Build (`make PR`) was run locally without failures

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: app/content/serializers/user.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 from ..models import Notification, Registration, User
 from .badge import BadgeSerializer
-from .event import EventInUserSerializer
+from .event import EventListSerializer
```
Comment (Her0elt) [inline app/content/serializers/user.py:6]: gidder du å gjøre dette til absolute imports

File: app/content/serializers/event.py [hunk 2]
```diff
@@ -2,16 +2,13 @@
...
+from app.content.serializers.priority import PrioritySerializer
 from app.util import EnumUtils
 
-from ..models import Event, Priority, Registration, User
-from .priority import PrioritySerializer
-
 
 class EventSerializer(serializers.ModelSerializer):
     expired = serializers.BooleanField(read_only=True)
-    registered_users_list = serializers.SerializerMethodField()
```
[the remark this answers is not in the document]
Comment (olros) [reply app/content/serializers/event.py:13]: Sjekk issuet ;) Men ja, trenger den ikke siden vi henter ut påmeldte gjennom et eget endepunkt som også er kun for admins

**Review by Her0elt — APPROVED** [review]
Big, looks good 😄

**Review by Eirsteir — APPROVED** [review]
Beautiful 🌮

**Pull request merged**